### PR TITLE
Cleanup of .rediff.com

### PR DESCRIPTION
**Pull request by spirillen**
## Summary

A lot of tracking records was listed/misplaced within the ads records.

Changes to be committed:
	modified:   abuse.txt
	modified:   ads.txt
	modified:   malware.txt
	modified:   tracking.txt

External relation:
  - https://mypdns.org/my-privacy-dns/matrix/-/issues/2137

Signed by @spirillen

## Checklist

- [X] I have verified that I have not modified any files inside the `alt-version` folder (automated code will automatically update those files)

- [X] I have verified that I have not modified any files inside the `dnsmasq-version` folder (automated code will automatically update those files)

### CODE REVIEW

File: malware.txt [hunk 1]
```diff
@@ -272042,7 +272041,7 @@
 0.0.0.0 track.prospertrack38.info
 0.0.0.0 track.publeadmedia.com
 0.0.0.0 track.qvod.com
-0.0.0.0 track.rediff.com
+
```
Comment (fishcharlie) [inline malware.txt:272044]: What is the purpose of this line?

**Review by fishcharlie — CHANGES_REQUESTED** [review]
I think there needs to be evidence that these domains don't belong on the other lists. Just because a domain is involved in tracking doesn't mean it can't be involved in malware, abuse, ads, etc.

For domains to be removed from lists there should be evidence provided that they don't belong on that list. Just because they are duplicated across lists is not evidence, and in fact can be purposeful.

File: malware.txt (hunk 1 above)
Comment (spirillen) [reply malware.txt:272044]: My counter question will then be:

Are you adding all records to all lists for which they could be categorized as, or only into the primary category?

That would be the next answer before I would be able to reply with a better answer as the number of available sources this somewhat limited

From the [hosts-sources](https://github.com/Import-External-Sources/hosts-sources) project

```shell
hosts-sources$ git grep 'rediff.com' 
data/blocklist_ads/domain.list:ads.rediff.com
data/blocklist_ads/domain.list:adworks.rediff.com
data/blocklist_ads/domain.list:datastore.rediff.com
data/blocklist_ads/domain.list:imads.rediff.com
data/blocklist_ads/domain.list:imadworks.rediff.com
data/blocklist_ads/domain.list:imworld.rediff.com
data/blocklist_ads/domain.list:indmetric.rediff.com
data/blocklist_ads/domain.list:ishare.metric.rediff.com
data/blocklist_ads/domain.list:loc.rediff.com
data/blocklist_ads/domain.list:metric.ind.rediff.com
data/blocklist_ads/domain.list:metric.rediff.com
data/blocklist_ads/domain.list:metricsishare.rediff.com
data/blocklist_ads/domain.list:newads.rediff.com
data/blocklist_ads/domain.list:shopmetric.rediff.com
data/blocklist_ads/domain.list:track.rediff.com
data/blocklist_ads/domain.list:usmetric.rediff.com
data/blocklist_malware/domain.list:imads.rediff.com
data/blocklist_malware/domain.list:imdownloads.rediff.com
data/blocklist_malware/domain.list:search.rediff.com
data/blocklist_malware/domain.list:server.toolbar.rediff.com
data/blocklist_malware/domain.list:track.rediff.com
data/disconnect-me/domain.list:ads.rediff.com
data/hphosts/ad_servers/domain.list:ads.rediff.com
data/hphosts/ad_servers/domain.list:adworks.rediff.com
data/hphosts/ad_servers/domain.list:imads.rediff.com
data/hphosts/ad_servers/domain.list:imadworks.rediff.com
data/hphosts/ad_servers/domain.list:track.rediff.com
data/hphosts/ad_servers/domain.list:metric.ind.rediff.com
data/hphosts/ad_servers/domain.list:ishare.metric.rediff.com
data/hphosts/emd/domain.list:imdownloads.rediff.com
data/hphosts/emd/domain.list:search.rediff.com
data/mitchellkrogza/Ultimate.Hosts.Blacklist/domain.list:ads.rediff.com
data/mitchellkrogza/Ultimate.Hosts.Blacklist/domain.list:adworks.rediff.com
data/mitchellkrogza/Ultimate.Hosts.Blacklist/domain.list:datastore.rediff.com
data/someonewhocares/domain.list:ads.rediff.com
data/someonewhocares/domain.list:track.rediff.com
data/yoyo.org/domain.list:ads.rediff.com
```

From [EasyList](https://github.com/easylist/easylist)

```shell
easylist$ git grep 'rediff.com'
easylist/easylist_allowlist_general_hide.txt:rediff.com#@#.ad_outer
easylist/easylist_specific_hide.txt:rediff.com###world_right1
easylist/easylist_specific_hide.txt:rediff.com###world_top
easyprivacy/easyprivacy_specific.txt:||indmetric.rediff.com^
easyprivacy/easyprivacy_specific.txt:||isharemetric.rediff.com^
easyprivacy/easyprivacy_specific.txt:||metricsishare.rediff.com^
easyprivacy/easyprivacy_specific.txt:||shopmetric.rediff.com^
easyprivacy/easyprivacy_specific.txt:||usmetric.rediff.com^
fanboy-addon/fanboy_annoyance_specific_hide.txt:rediff.com###div_most_commented
fanboy-addon/fanboy_annoyance_specific_hide.txt:rediff.com###span_subscibe
fanboy-addon/fanboy_annoyance_specific_hide.txt:rediff.com###div_subscribe_in_top
```

**Comment by blocklistproject** [comment]
I will look this over this evening.

**Pull request merged**